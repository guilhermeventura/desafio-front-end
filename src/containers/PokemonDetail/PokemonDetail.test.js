import React from "react";
import { Provider } from "react-redux";
import configureMockStore from "redux-mock-store";
import thunk from "redux-thunk";
import renderer from "react-test-renderer";

import { default as PokemonDetail } from "./index";

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);
const mockData = {
  params: {
    id: 1
  }
};
test("renders correctly", () => {
  const store = mockStore({ pokemons: [] });
  const component = renderer
    .create(
      <Provider store={store}>
        <PokemonDetail match={mockData} />
      </Provider>
    )
    .toJSON();

  expect(component).toMatchSnapshot();
});
