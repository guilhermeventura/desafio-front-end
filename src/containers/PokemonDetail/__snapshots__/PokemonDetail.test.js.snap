// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="columns is-multiline"
  style={Object {}}
>
  <div
    className="column"
    style={Object {}}
  />
</div>
`;
